flake.nix

```diff
@@ -40,7 +39,7 @@
           overlays = [solc-pkgs.overlay];
           config = { allowBroken = true; };
         });
-        solc = (solc-pkgs.mkDefault pkgs pkgs.solc_0_8_24);
+        solc = (solc-pkgs.mkDefault pkgs pkgs.solc_0_8_26);
```

Comment:
if the solc required for tests changes, please update it on `.github/workflows/build.yml` for the Windows job as well 👍

Reply:
Ooops, thanks, I just did. It's a huge mess with nix right now anyway, so everything is blocked...

Reply:
@msooseth Looking at the build failures, maybe bumping the `solc-pkgs` overlay is enough to get 0.8.26 here? The latest version should include it.

Reply:
Sorry, nix is absolutely a huge mystery to me. Can you maybe propose a fix? Note that @arcz has a nice PR update to `flake.nix`: https://github.com/ethereum/hevm/pull/548

Reply:
arcz bumped it now :) it looks like the tests got to run on Ubuntu; there's many test failures though https://github.com/ethereum/hevm/actions/runs/10701065518/job/29666189369?pr=529